kDriveCore/Data/UploadQueue/Queue/UploadQueue.swift

```diff
@@ -139,6 +139,10 @@ public final class UploadQueue: ParallelismHeuristicDelegate {
             let isSuspended = (shouldSuspendQueue || forceSuspendQueue)
             operationQueue.isSuspended = isSuspended
             Log.uploadQueue("observeNetworkChange :\(isSuspended)")
+            if ReachabilityListener.instance.currentStatus == .wifi || !UserDefaults.shared.isWifiOnly {
+                self.cleanNetworkAndLocalErrorsForAllOperations()
+                self.resumeAllOperations()
```

Comment:
Here we should be able to get away with mutating `operationQueue.isSuspended` instead of calling directly the rebuild queue methods.